mms/service.py

```diff
@@ -60,20 +60,22 @@ def retrieve_data_for_inference(batch):
             raise ValueError("Received invalid inputs")
 
         req_to_id_map = {}
-
+        headers = dict()
         input_batch = []
         for batch_idx, request_batch in enumerate(batch):
             req_id = request_batch.get('requestId').decode("utf-8")
             parameters = request_batch['parameters']
+            model_in_headers = dict()
 
             model_in = dict()
             for parameter in parameters:
                 model_in.update({parameter["name"]: parameter["value"]})
-
+                model_in_headers.update({parameter["name"]: {"content-type": parameter["contentType"]}})
```

Comment:
Small nit here : 
Should the parameter key be ```content-type``` and not ```contentType``` ?

Reply:
`contentType` is what we get from a message from frontend. I think we can leave it as is. It is easy for us to differentiate between the message received from frontend and the actual metadata that we should use in multiple parts of the code.